buildscripts/download-jars.sh

```diff
@@ -7,8 +7,8 @@
 # https://github.com/jfrog/maven-dep-tree
 
 # Once you have updated the versions mentioned below, please execute this script from the root directory of the jfrog-cli-core to ensure the JAR files are updated.
-GRADLE_DEP_TREE_VERSION="3.0.0"
-MAVEN_DEP_TREE_VERSION="1.0.0"
+GRADLE_DEP_TREE_VERSION="3.0.1"
+MAVEN_DEP_TREE_VERSION="1.0.2"
```

Comment:
Let's also add that changing the MAVEN_DEP_TREE_VERSION here requires also a change in xray/commands/audit/sca/java/mvn.go